docs/en/configuration.md

```diff
@@ -28,3 +28,31 @@ DMSDocumentAddController:
     - php
     - php5
 ```
+
+## Adding fields to the Query Builder ##
```

Comment:
We don't normally add the trailing heading marks, there have been times in the past where our markdown renderers in the docs sites don't handle this correctly - could you remove them?